README.md

```diff
@@ -196,6 +196,7 @@ A mocked MMKV instance is automatically used when testing with Jest, so you will
 * [Using MMKV with mobx-persist](./docs/WRAPPER_MOBXPERSIST.md)
 * [Using MMKV with zustand persist-middleware](./docs/WRAPPER_ZUSTAND_PERSIST_MIDDLEWARE.md)
 * [Using MMKV with jotai](./docs/WRAPPER_JOTAI.md)
+* [Using MMKV with react-query](./docs/WRAPPER_REACTQUERY.md)
```

Comment:
Can you rename the file to `WRAPPER_REACT_QUERY.md` (`_` between REACT and QUERY)?